config/initializers/secure_headers.rb

```diff
@@ -14,11 +12,17 @@
   config.x_content_type_options = "nosniff"
 
   # http://msdn.microsoft.com/en-us/library/dd565647%28v=vs.85%29.aspx
-  config.x_xss_protection = { :value => 1 }
+  config.x_xss_protection = "1; mode=block"
```

Comment:
We're not matching the previous behaviour here.

The default value has changed between [3.0](https://github.com/twitter/secureheaders/blob/master/lib/secure_headers/headers/x_xss_protection.rb#L5) and [2.0](https://github.com/twitter/secureheaders/blob/2.x/lib/secure_headers/headers/x_xss_protection.rb#L6). Don't know if we should be using the new default here.